try-tidb.md

```diff
@@ -1,12 +1,14 @@
 ---
 title: Try TiDB
-summary: Some examples on how to test a TiDB cluster.
+summary: Basic SQL statement for the TiDB database.
```

Comment:
For the "summary", we have been using a sentence pattern starting with a verb. So to keep it consistent, it'd be better to update it to: 

**Learn about the** basic SQL **statements** for the TiDB database.